src/StockportGovUK.NetStandard.Gateways.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
     <Title>A package that provides base gateways using HttpClient</Title>
-    <VersionPrefix>6.0.2</VersionPrefix>
+    <VersionPrefix>6.1.0</VersionPrefix>
```

Comment:
If this is a patch bump in a dependency then it would be a patch bump here as well not a minor